packages/next/client/image.tsx

```diff
@@ -210,21 +220,35 @@ export default function Image({
   unoptimized = false,
   priority = false,
   loading,
-  layout,
   className,
   quality,
   width,
   height,
-  unsized,
-  ...rest
+  ...all
 }: ImageProps) {
   const thisEl = useRef<HTMLImageElement>(null)
 
+  let rest: Partial<ImageProps> = all
+  let layout: NonNullable<LayoutValue> = sizes ? 'responsive' : 'intrinsic'
+  if ('unsized' in rest) {
+    // Unsized images default to the fill layout mode:
+    layout = 'fill'
```

Comment:
Let's not break existing pages, unsized should work as it did before

Reply:
Done